books/free-programming-books-es.md

```diff
@@ -205,7 +205,8 @@
 ### JavaScript
 
 * [El Tutorial de JavaScript Moderno](https://es.javascript.info) - Ilya Kantor, Elizabeth Portilla, joaquinelio, Ezequiel Castellanos, et al. (HTML)
-* [Eloquent JavaScript](https://eloquentjs-es.thedojo.mx) - Marijn Haverbeke, `trl.:` Various (HTML, PDF, EPUB, MOBI)
+* [Eloquent JavaScript (3ra Edición)](https://eloquentjs-es.thedojo.mx) - Marijn Haverbeke, `trl.:` Various (HTML, PDF, EPUB, MOBI)
+* [Eloquent JavaScript (4ta Edición)](https://www.eloquentjavascript.es) - Marijn Haverbeke (HTML) [(PDF)](https://github.com/GNUXDAR/js-de-0-a-experto/blob/master/Docs/EbookEloquent/Eloquent_JavaScript_4ta_ES.pdf)
```

Comment:
```suggestion
* [Eloquent JavaScript (4ta Edición)](https://www.eloquentjavascript.es) - Marijn Haverbeke (HTML) (PDF)
```
There is already a link to the PDF on the eloquentjavascript.es page. extra links create a mainenance burden for us.

Reply:
Ok, i go update commit